pkg/efscreate/efs.go

```diff
@@ -217,7 +218,7 @@ func (efs *EFS) waitForAvailableMountTarget() error {
 	efsID := efs.resources.efsID
 	describeInput := &awsefs.DescribeMountTargetsInput{FileSystemId: aws.String(efsID)}
 	backoff := wait.Backoff{
-		Duration: operationDelay,
+		Duration: volumeCreateInitialDelay,
 		Factor:   operationBackoffFactor,
```

Comment:
I made this change because - I noticed waiting for mount target creation timing out in some cases and this just bumps up the timeout duration.